libs/platform/src/lib/components/table/components/dialogs/filtering/filter-step.component.ts

```diff
@@ -0,0 +1,61 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    EventEmitter,
+    forwardRef,
+    Input,
+    Output,
+    TemplateRef,
+    ViewChild
+} from '@angular/core';
+
+import { FilterType } from '../../../enums';
+import { CollectionFilter } from '../../../interfaces';
+import { TableViewSettingsFilterComponent } from '../../table-view-settings-filter/table-view-settings-filter.component';
+import { FiltersViewStep, FILTERS_VIEW_STEP_TOKEN } from './filters-active-step';
+
+@Component({
+    selector: 'fdp-filter-step',
+    templateUrl: './filter-step.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [{ provide: FILTERS_VIEW_STEP_TOKEN, useExisting: forwardRef(() => FilterStepComponent) }]
+})
+export class FilterStepComponent implements FiltersViewStep {
+    @Input()
```

Comment:
Shouldn't there(and in rest of files) be some comments claryfing inputs/properties? I see it's used internally, but still

Reply:
Added comments for some properties.